scripts/apps/workspace/content/views/profile-settings.html

```diff
@@ -90,7 +90,7 @@ <h3 class="modal__heading">{{ :: 'Editing' | translate }} "{{ editing.form.label
     </div>
     <div class="modal__body">
         <form name="editForm">
-            <fieldset>
+            <fieldset ng-init="setNgForm(editForm)">
```

Comment:
not sure this is needed, the form should be available on scope as `editForm`

Reply:
Accessing it from scope [isn't working](https://user-images.githubusercontent.com/2076953/51896550-044d4d00-23ad-11e9-8002-39c7450e452e.png)

Reply:
:ok: